include/boost/geometry/algorithms/detail/buffer/turn_in_piece_visitor.hpp

```diff
@@ -1,831 +1,46 @@
 // Boost.Geometry (aka GGL, Generic Geometry Library)
 
-// Copyright (c) 2012-2014 Barend Gehrels, Amsterdam, the Netherlands.
-// Copyright (c) 2017 Adam Wulkiewicz, Lodz, Poland.
-
-// This file was modified by Oracle on 2016, 2018.
-// Modifications copyright (c) 2016-2018 Oracle and/or its affiliates.
-// Contributed and/or modified by Adam Wulkiewicz, on behalf of Oracle
+// Copyright (c) 2020 Barend Gehrels, Amsterdam, the Netherlands.
```

Comment:
This file is for 99% replaced (I started with a new file).
So I removed the Oracle copyright, but if you want to keep it, OK for me.

Reply:
I have no opinion. I'm not a legal expert. :)
I'd rather worry about files to which you copied the code from this one because I see a lot of changes done by me regarding the propagation of strategies and one change done by Menelaos in the past so technically if you copied any part affected by us into a different file this file should have all of the copyrights too I think. But this is only my guess.

Reply:
When I view this version of the file and click "Blame" I still see some changes made by me and Menelaos so even though the majority of the code was removed our changes are still there. This one even adds my copyrights: https://github.com/boostorg/geometry/commit/fd72a1aa16a71ac465db334890964a3799bf5686
Not sure why, though, maybe I changed something during weekend ;)

Reply:
> When I view this version of the file and click "Blame" I still see some changes made by me and Menelaos, see:

No problem, I'll put it back.